src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -1326,19 +1326,22 @@ int fx_muxer_t::read_config_and_execute(
         return rc;
     }
 
-    auto config = app->get_runtime_config();
+    auto app_config = app->get_runtime_config();
 
     // 'Roll forward on no candidate fx' is set to 1 (roll_fwd_on_no_candidate_fx_option::minor) by default. It can be changed through:
-    // 1. Command line argument (--roll-forward-on-no-candidate-fx). Only defaults the app's config.
+    // 1. Command line argument (--roll-forward-on-no-candidate-fx).
     // 2. Runtimeconfig json file ('rollForwardOnNoCandidateFx' property), which is used as a default for lower level frameworks if they don't specify a value.
-    // 3. DOTNET_ROLL_FORWARD_ON_NO_CANDIDATE_FX env var. Only defaults the app's config.
-    // The conflicts will be resolved by following the priority rank described above (from 1 to 3).
+    // 3. Runtimeconfig json file ('rollForwardOnNoCandidateFx' property in "framework" section:).
```

Comment:
I would think the order between 2 & 3 would be flipped.  The 'rollForwardOnNoCandidateFx' property in the "framework" section should 'win' over the global one.

Reply:
OK thanks. I will fix the comment - the comment is wrong, but the current semantics are what you describe.